src/OrbitGl/GlCanvas.h

```diff
@@ -89,23 +89,23 @@ class GlCanvas {
   virtual std::vector<std::string> GetContextMenu() { return std::vector<std::string>(); }
```

Comment:
and probably here?

Reply:
Thanks again, I also added a third one, sorry.